### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rajuramachandra**
Please review the code properly

### CODE REVIEW

**Comment by rajuramachandra** [comment]
# # Please someone help me

**Comment by rajuramachandra** [comment]
> # # Please someone help me

I'll help you

**Review by rajuramachandra — COMMENTED** [review]
@hemanth986 please work on this

**Review by ghost — COMMENTED** [review]
heLLooooo

**Review by ghost — COMMENTED** [review]
#15  nn

**Review by ghost — COMMENTED** [review]
hhhhh

File: inventory.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 Target1  ansible_host=172.17.0.2  ansible_user = root
 Target2  ansible_host=172.17.0.3  ansible_user = root
```
Comment (ghost) [inline inventory.txt:4]: hhdkjdk

Comment (ghost) [inline inventory.txt:4]: hhh

**Review by ghost — APPROVED** [review]

**Pull request merged**